content/docs/other-guides/support.md

```diff
@@ -0,0 +1,168 @@
+
+Consult the support page for the cloud service that you're using:
+
+* [Amazon Web Services (AWS)](https://aws.amazon.com/contact-us/)
+* [Google Cloud Platform (GCP)](https://cloud.google.com/support-hub/)
+* [IBM Cloud Private](https://www.ibm.com/cloud/support)
+* [Microsoft Azure](https://azure.microsoft.com/en-au/support/options/)
+
+## Support from the Kubeflow community
+
+Kubeflow has an active and helpful community of users and contributors. You
+can ask questions and make suggestions in the following places:
+
+* **Slack** for online chat and messaging. See details of Kubeflow's 
+  [Slack workspace and channels](/docs/about/community/#slack).
+
+* **Kubeflow discuss** for email-based group discussion. Join the
+  [kubeflow-discuss](https://groups.google.com/forum/#!forum/kubeflow-discuss) 
+  group.
+
+* **Kubeflow documentation** for overviews and how-to guides. In particular,
+  refer to the following documents when troubleshooting a problem:
+
+  * [Kubeflow installation and setup](/docs/started/getting-started/)
+  * [Kubeflow components](/docs/components/)
+  * [Further setup and troubleshooting](/docs/other-guides/)
+
+* **Kubeflow issue trackers** for known issues, questions, and feature requests.
+  Search the open issues to see if someone else has already logged the problem 
+  that you're encountering and learn about any workarounds to date. If no-one
+  has logged your problem, create a new issue to describe the problem.
+
+    Each Kubeflow application has its own issue tracker within the [Kubeflow
+    organization on GitHub](https://github.com/kubeflow). To get you started,
+    here are the primary issue trackers:
+
+  * [Kubeflow core](https://github.com/kubeflow/kubeflow/issues)
+  * [kfctl command-line tool](https://github.com/kubeflow/kfctl/issues)
+  * [Kustomize manifests](https://github.com/kubeflow/manifests/issues)
+  * [Kubeflow Pipelines](https://github.com/kubeflow/pipelines/issues)
+  * [Katib hyperparameter tuning](https://github.com/kubeflow/katib/issues)
+  * [Metadata](https://github.com/kubeflow/metadata/issues)
+  * [Fairing notebook SDK](https://github.com/kubeflow/fairing/issues)
+  * [TensorFlow training (TFJob)](https://github.com/kubeflow/tf-operator/issues)
+  * [PyTorch training (PyTorchJob)](https://github.com/kubeflow/pytorch-operator/issues)
+  * [KFServing](https://github.com/kubeflow/kfserving/issues)
+  * [Examples](https://github.com/kubeflow/examples/issues)
+  * [Documentation](https://github.com/kubeflow/website/issues)
+
+## Support from partners
+
+TODO Shall we include a partners section like this?
+
+The following organizations offer advice and support for Kubeflow deployments:
+
+* [Arrikto](https://www.arrikto.com). Contacts: TODO We need a point of contact.
+  Should we list some or all of the contacts that 
+  are currently in the [org list](https://github.com/kubeflow/community/blob/master/member_organizations.yaml#L42)?
+
+* TODO Are there any other partners we should add?
```

Comment:
Noting the TODO.

@jbottum and @abhi-g and @theadactyl Do you have any input here?

Reply:
@sarahmaddox My initial thought would be to figure out what the table should contain and then ask anyone who wants to be listed to submit the PR themselves so its clear it came from them.

This might lead to duplicaiton of info here and in member_organizations but that's probably less of an immediate concern and we can resolve that later.

Reply:
Good idea. Done.